src/apps/applets/crud/applets.py

```diff
@@ -226,3 +226,13 @@ async def delete_access_link(self, applet_id: int):
         query = query.where(AppletSchema.id == applet_id)
         query = query.values(link=None, require_login=None)
         await self._execute(query)
+
+    async def transfer_ownership(
+        self, applet_id: int, new_owner_id: int
+    ) -> AppletSchema:
+        query: Query = update(AppletSchema)
+        query = query.where(AppletSchema.id == applet_id)
+        query = query.values(account_id=new_owner_id)
+        query = query.returning(self.schema_class)
+        db_result = await self._execute(query)
+        return db_result.scalars().one_or_none()
```

Comment:
There the return value could be `None` also

Reply:
I would also say that returning None from here make no sense, so then raising some error is a good idea IMHO